lib/job/arExportJob.class.php

```diff
@@ -364,4 +367,19 @@ protected function getUniqueFilename($filepath)
       $pathinfo['extension']
     );
   }
+
+  /**
+   * Log export progress every LOG_INTERVAL rows
```

Comment:
Nitpick: maybe add the cache clearing to the description of the function.

Reply:
Done!